plugins/stack-overflow/config.d.ts

```diff
@@ -24,5 +24,11 @@ export interface Config {
      * @visibility frontend
      */
     baseUrl?: string;
+
+    /**
+     * The api key to authenticate to Stack Overflow API
+     * @visibility frontend
```

Comment:
Oh no! This should absolutely not need to be the case - we explicitly want to keep secret things out of the frontend. It should rather be marked as `@visibility secret` and the frontend should not try to access it.

Reply:
this is why I stick to backend work haha. I am going to remove the frontend work for now and just get backend changes in.

Reply:
I can contribute the other stuff @mmulligan03 so that we can get this merged, thanks a lot for you time!